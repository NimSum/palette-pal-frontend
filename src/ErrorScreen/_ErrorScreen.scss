#ErrorScreen,
.loading-screen {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: calc(100vh - 130px);

  img {
    border-radius: 100%;
    margin-bottom: 20px;
  }
}