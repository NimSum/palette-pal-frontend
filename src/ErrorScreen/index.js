import React from 'react';
import './_ErrorScreen.scss';

const ErrorScreen = () => {
  return (
    <div id="ErrorScreen">
      <h2>404 Page Not Found :(</h2>
    </div>
  );
};

export default ErrorScreen;
