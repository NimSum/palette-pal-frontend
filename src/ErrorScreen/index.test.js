import React from 'react';
import ErrorScreen from './index';
import { shallow } from 'enzyme';

describe('ErrorScreen', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(< ErrorScreen />);
  })
  
  it('shoud match component ErrorScreen snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})