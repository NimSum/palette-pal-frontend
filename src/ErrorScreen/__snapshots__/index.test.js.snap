// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen shoud match component ErrorScreen snapshot 1`] = `ShallowWrapper {}`;
